### PR TITLE
63 single error message

### PR DESCRIPTION
**Pull request by aMotherFox**

### CODE REVIEW

File: src/main/java/com/my/FoodTruckBackend/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class HelloWorld {
+    private String message;
+}
```
Comment (danielKNS) [inline src/main/java/com/my/FoodTruckBackend/HelloWorld.java]: I don't think this is being used. Is it needed ?

File: src/main/java/com/my/FoodTruckBackend/HelloWorldController.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/helloWorld")
+    public String helloWorld() {
+        return "HELLO WORLD!!!";
+    }
+}
```
Comment (danielKNS) [inline src/main/java/com/my/FoodTruckBackend/HelloWorldController.java]: I don't think this API is being used. Is it needed ?

File: src/main/java/com/my/FoodTruckBackend/customer/signup/CustomerService.java [hunk 3]
```diff
@@ -21,10 +21,39 @@ public List<CustomerDto> getListOfCustomers() {
...
+                "Please enter a unique password, between 2-50 characters"
+            );
+        }
+        if (
+            newCustomerRequestBody.getConfirmPassword().length() < 2 ||
+            newCustomerRequestBody.getConfirmPassword().length() > 50
+        ) {
+            throw new ResponseStatusException(
+                HttpStatus.BAD_REQUEST,
+                "Please re-enter your unique password, between 8-50 characters"
```
Comment (danielKNS) [inline src/main/java/com/my/FoodTruckBackend/customer/signup/CustomerService.java]: On line 49 is the password between 8-50 or 2-50 ?

File: src/main/java/com/my/FoodTruckBackend/HelloWorld.java (hunk 1 above)
Comment (aMotherFox) [reply src/main/java/com/my/FoodTruckBackend/HelloWorld.java]: This was only for testing and has been removed

File: src/main/java/com/my/FoodTruckBackend/HelloWorldController.java (hunk 2 above)
Comment (aMotherFox) [reply src/main/java/com/my/FoodTruckBackend/HelloWorldController.java]: This was only for testing and has been removed

File: src/main/java/com/my/FoodTruckBackend/customer/signup/CustomerService.java (hunk 3 above)
Comment (aMotherFox) [reply src/main/java/com/my/FoodTruckBackend/customer/signup/CustomerService.java]: this has been fixed

**Review by vportela — APPROVED** [review]

**Review by danielKNS — APPROVED** [review]

**Pull request merged**